src/std_clock.h

```diff
@@ -19,15 +19,11 @@
 #define EP_STD_CLOCK_H
 
 #include <chrono>
-#include <type_traits>
 #include <thread>
 
 #if !defined(_3DS) && !defined(__SWITCH__) && !defined(PSP2)
 struct StdClock {
-	using clock = std::conditional_t<
-		std::chrono::high_resolution_clock::is_steady,
-		std::chrono::high_resolution_clock,
-		std::chrono::steady_clock>;
+	using clock = std::chrono::steady_clock;
```

Comment:
Looks like the logic about high res clock can be removed in `Name()` function as well?